### ISSUE TITLE
Set min property dynamically

### ISSUE THREAD

**Issue by dgroh** [vlio20/angular-datepicker#269]
How do I set the `min` property dynamically?

I have a **Date from** and a **Date to** - The `min` of **Date to** must be the selected value of **Date from** I tried to achieve this by setting the `component.config.min` but nothing happens. 

This stackblitz demonstrates this: [Change min date dynamically](https://stackblitz.com/edit/angular-trty1e)

**Comment by vlio20** [comment]
In order to change the configuration (and min is part of it) you need to pass a new reference to `this.dateToDp.config`:

```
    this.filterForm.get("dateFrom").valueChanges.subscribe(value => {
      console.log(this.dateToDp);
      this.dateToDp.config = {
        min: value,
        ...this.dateToDp
      };
    });
  }
``` 

here is a working stackblitz: https://stackblitz.com/edit/angular-3h4a7c

**Comment by dgroh** [comment]
The rule of showing the `min` date when the picker opens should be applied, too. Means, the  picker should start from the `min` date, what is not happening when changing the configuration.

**Comment by miyolou** [comment]
You can bind the value changes from 'Date from' to the displayDate attr on the 'Date to' input element.

**Comment by dgroh** [comment]
@miyolou yes, this would be a workaround tho. this feature has been implemented in 2.6.0, so this should work also in this scenario imo, but yes, for now I'm using displayDate as workaorund.

**Comment by dgroh** [comment]
it's also not working with `displayDate`: https://stackblitz.com/edit/angular-rxrmxw

**Comment by dgroh** [comment]
@vlio20 using `...this.dateToTop` is overriding the format configuration, note that the format is `DD.MM.YYYY` but fater overriding it it gets `MM-MM-YYYY`

**Comment by vlio20** [comment]
@dgroh, here is a beter configuration: https://stackblitz.com/edit/angular-3h4a7c?file=app%2Fapp.component.ts

There is still some issue with the min date in the "to" calendar.

**Comment by dgroh** [comment]
Thanks for your help

**Comment by vlio20** [comment]
resolved in 969eb01

**Comment by vlio20** [comment]
Please open a new ticket. It should not be changed. Please put a new stackblitz there, with latest version
